Reflective Report.tex

```diff
@@ -0,0 +1,85 @@
+
+\section{Introduction}
+
+The three weakness I'll be exploring in this report are: time management, critical evaluation, . All of these skills are significant to me for professional practise and have been emphasised during my first semester.
+
+\section{Time Management}
+
+\subsection{Description - What happened? What is being examined?}
+
+One of my weaknesses was time management. With most tasks, it is crucial to follow a strict and thoughtful plan which accounts for all aspects of the project. For me, the relevance of this issue was for not anticipating all aspects of each project.
+
+\subsection{How has this impacted my learning and projects?}
+
+I found that my allocated work time had been misplaced. For me, most of the time was focused on research of ideas and techniques rather than practical applications. This not only lead to a low quality of work but also low quantity too. This uneven distribution also hindered my overall learning.
+
+
+\subsection{Interpretation - What is relevant about this idea?}
+
+Initially, I noticed that I did not feel confident in my abilities and questioned what I could do to improve. For me, this meant that I enjoyed the course much less. At the time, I did not know how to overcome this issue which lead to an endless cycle of frustration and research.
+
+\subsection{Outcome- What have I learned and what can be done in the future?}
+
+I now realise I should have notified my lecturers of my struggles. I have significantly changed my opinion of how programming works, and my knowledge of self teaching has improved. For the future, I aim to make strict plans on how to structure my time and for how long one task is expected to take. If the task is not completed, then I must approach my lecturer or colleagues with my concerns.
+
+
+\section{Critic Ability}
+
+\subsection{Description - What happened? What is being examined?}
+
+Another of my weaknesses was critical evaluation, which requires the objective review of both essays and coding work. It has been crucial to evaluate both my work and my peers. The ability to critique your own work allows for issues to be discussed and improvements to be made.
+
+\subsection{How has this impacted my learning and projects?}
+
+Unlike my previous education, these tasks required making my own objective evaluation of work quality. This lead to being over critical of my own work, in a way that wasn't constructive.
+
+\subsection{Interpretation - What is relevant about this idea?}
+
+For me, the significant issue arose from very low confidence. Subsequently, I felt I was not appropriately equipped to offer critical advice. In addition to this, I felt unfamiliar with marking others work with regards to coding practise.
+
+\subsection{Outcome- What have I learned and what can be done in the future?}
+
+Having applied this skill over several review sessions, I now realise that, while making reference to project objectives, evaluation is more structured and focused.
```

Comment:
:+1: --- This is going in the right direction, but needs to be further developed. The action should be specific and measurable.